app/internal/email.py

```diff
@@ -1,11 +1,27 @@
+import os
+from typing import List, Optional
+
+from starlette.templating import Jinja2Templates
+
 from app.config import email_conf
 from app.database.models import Event, User
-from fastapi import BackgroundTasks
+from fastapi import BackgroundTasks, UploadFile
 from fastapi_mail import FastMail, MessageSchema
+from pydantic import EmailStr
+from pydantic.errors import EmailError
 from sqlalchemy.orm.session import Session
 
 mail = FastMail(email_conf)
 
+templates = Jinja2Templates(directory=os.path.join("app", "templates"))
```

Comment:
Can you please move this, and the other 3 constant variables after this line, to the config file?

Reply:
ok :)